Sources/Extensions/SwiftStdlib/StringExtensions.swift

```diff
@@ -484,6 +484,22 @@ public extension String {
     }
     #endif
 
+    /// SwifterSwift: Returns the dominant language of `String`
```

Comment:
Apple's documentation generally refers to `self` as "the receiver".
```suggestion
    /// SwifterSwift: Returns the dominant language of the receiver
```

Reply:
Done.